fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -857,8 +1103,7 @@ private static JoinedRecordTypeBuilder joinedMetadataHook(@Nonnull final RecordM
         final JoinedRecordTypeBuilder joined = metaDataBuilder.addJoinedRecordType("luceneJoinedPartitionedIdx");
         joined.addConstituent("complex", "ComplexDocument");
         joined.addConstituent("simple", "SimpleDocument");
-        joined.addJoin("simple", field("group"), "complex", field("group"));
-        metaDataBuilder.addIndex("SimpleDocument", "simple$group", field("group"));
+        joined.addJoin("simple", field("doc_id"), "complex", field("header").nest("header_id"));
```

Comment:
Having the grouping field also be the join field made it nearly impossible to do multi-group tests that made sense, and I don't think there's actually a use case for that.